dist/stat_data.py

```diff
@@ -539,6 +539,16 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Live Restore statistics
     ##########################################
+    LiveRestoreStat('live_restore_bytes_copied', 'the number of bytes copied from the source to the destination'),
```

Comment:
All stat descriptions should start with "live restore" so they are grouped together in t2 and are easily searchable.

Reply:
The LiveRestoreStat takes care of this for us: 
![image](https://github.com/user-attachments/assets/81b66b1a-7d7d-470c-b851-4fa3969909aa)